src/webserver/webcache/TimedObject.java

```diff
@@ -0,0 +1,93 @@
+package webserver.webcache;
+
+/**
+ * A timed class that holds an object, as well as an associated expiry time.
+ * <p>
+ * After the current system time as retrieved by
+ * {@code System.currentTimeMillis()} is past than the expiry time, this class
+ * object will have expired and can be acted upon.
+ * <p>
+ * Created <b> 2021-01-08</b>.
+ * 
+ * @since 0.0.1
+ * @version 0.0.1
+ * @author Joseph Wang
+ * 
+ */
+public class TimedObject<T> {
```

Comment:
Not really, besides that others can... extend these for their own purposes?

It does make sense to just combine this and Node anyways since this is already in the webcache package and only the webcache uses it.